src/layouts/PublicVuetify.vue

```diff
@@ -48,49 +54,48 @@ import navItems from '@/data/navigation';
 
 @Component
 export default class PublicVuetify extends Vue {
-  private items = navItems;
-  private drawer: boolean | null = null;
+	private items = navItems;
```

Comment:
Seems like you need to configure your IDE properly: this should be indented with 2 spaces, not a tab.